src/hotspot/share/classfile/systemDictionary.cpp

```diff
@@ -782,15 +781,15 @@ InstanceKlass* SystemDictionary::resolve_instance_class_or_null(Symbol* name,
           if (oldprobe->check_seen_thread(THREAD, PlaceholderTable::LOAD_INSTANCE)) {
             throw_circularity_error = true;
           } else {
-            // case 1: traditional: should never see load_in_progress.
+            // case 3: traditional: should never see load_in_progress.
```

Comment:
It seems odd to hit case 3 first in the code. I think the comment ordering of the cases should be retained.

Reply:
I seriously disliked the comment case ordering.  It started with a special case and finally hit the general case. This is why I've never read this comment before or it never made any sense to me. Now it does.